### DISCUSSION TITLE
How can I see my total commits?

### DISCUSSION THREAD

**Discussion by MitkoVtori** [anuraghazra/github-readme-stats#2272] [Q&A] [answered]
Hi, can someone tell me how can I see my total commits? 
Right now, I see my total commits(2022), I don't want to see the commits of a current year, I want to see all my commits.
### Here's how my stats look like
![Screenshot 2022-11-20 163628](https://user-images.githubusercontent.com/112943652/202908374-505ea4a5-4092-4a56-8d05-761f87c7420e.png)
### Theme: aura

**Comment by rickstaa** [comment, marked as answer]
@MitkoVtori Thanks for your answer you can show all commits using the `include_all_commits` option (see https://github.com/anuraghazra/github-readme-stats#stats-card-exclusive-options).

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=MitkoVtori&include_all_commits=true&theme=aura)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=MitkoVtori&include_all_commits=true&theme=aura)](https://github.com/anuraghazra/github-readme-stats)
```

**Reply by MitkoVtori** [reply]
Thank you!

**Comment by Victorcorcos** [comment]
Thank you! I was looking for that:

```rb
&include_all_commits=true
```

![Victor's GitHub stats](https://github-readme-stats2-neon.vercel.app/api?username=Victorcorcos&show=prs_merged,prs_merged_percentag&show_icons=true&include_all_commits=true)

**Comment by mohamadmahdi-87** [comment]
Why doesn't it work?
